.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,16 @@
+name: pre-commit
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v3
+    - uses: pre-commit/action@v3.0.0
```

Comment:
It looks like using `pre-commit/action` lead to double cloning NNCF repository.

Reply:
I remove pre-commit to use make instead. 
But no, fetch repository happens only on checkout step. https://github.com/openvinotoolkit/nncf/actions/runs/4630679783/jobs/8192572026